validation-test/stdlib/POSIXErrorCode.swift

```diff
@@ -0,0 +1,67 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+//
+// REQUIRES: OS=macosx
+// REQUIRES: OS=ios
+// REQUIRES: OS=tvos
+// REQUIRES: OS=watchos
+// REQUIRES: OS=linux-androideabi
+// REQUIRES: OS=linux-gnu
+
+import Swift
+import StdlibUnittest
+
+#if os(Linux) || os(Android)
+import Glibc
+#elseif os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Darwin
+#endif
+
+var POSIXErrorCodeTestSuite = TestSuite("POSIXErrorCode")
+
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+
+POSIXErrorCodeTestSuite.test("Darwin POSIX error codes constants") {
+    
+  expectEqual(POSIXErrorCode.EPERM.rawValue, EPERM)
+
+#endif
+
+#elseif os(Linux) || os(Android)
+
+POSIXErrorCodeTestSuite.test("Linux POSIX error codes constants") {
+    
+  expectEqual(POSIXErrorCode.EPERM.rawValue, EPERM)
```

Comment:
Sorry, one more thing: the syntax of `expectEqual` is `expectEqual(expected, actual)` so the checks should be following the pattern of `expectEqual(EPERM, POSIXErrorCode.EPERM.rawValue)`.